docker-cloud/dockerid.md

```diff
@@ -25,7 +25,10 @@ Bitbucket from your Docker Cloud account settings.
 
 ## Email addresses
 
-You can associate multiple email addresses with your Docker ID, and one of these addresses becomes the primary address for the account. The primary address is used by Docker to send password reset notifications and other important information, so be sure to keep it updated.
+You can associate multiple email addresses with your Docker ID, and one of these
+addresses becomes the primary address for the account. The primary address is
```

Comment:
s/and one of these addresses/and one of these/

(Just to avoid saying "addresses" 3x)